chainindex/api.go

```diff
@@ -57,9 +57,10 @@ func (si *SqliteIndexer) getTipsetCountsAtHeight(ctx context.Context, height abi
 }
 
 func (si *SqliteIndexer) ChainValidateIndex(ctx context.Context, epoch abi.ChainEpoch, backfill bool) (*types.IndexValidation, error) {
-	if !si.started {
-		return nil, xerrors.Errorf("ChainValidateIndex can only be called after the indexer has been started")
+	if !si.started && si.closed {
```

Comment:
You need to take a read lock before reading this. See how we do it elsewhere.

Reply:
Updated.
Added a function `isClosed`, so we can reuse it.